manim/mobject/stats.py

```diff
@@ -0,0 +1,129 @@
+r"""Mobjects that have to do with statistics and statistical analysis.
+
+Examples
+--------
+.. manim:: ShowHistogram
+    :save_last_frame:
+
+    class ShowHistogram(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            histogram = Histogram([0, 1, 2, 3, 4], [5, 1, 4, 2, 3], self)
+
+            self.add(histogram)
+
+"""
+
+import numpy as np
+from ..mobject.functions import ParametricFunction
+from ..scene.graph_scene import GraphScene
+from ..constants import *
+
+__all__ = [
+    "Histogram",
+]
+
+
+class Histogram(ParametricFunction):
+    """A Histogram VMobject (optionally scaled to a graph) that can be used to
+    easily show statistical data.
+
+    Parameters
+    ----------
+    x : :class:`list`, :class:`tuple`, :class:`numpy.ndarray`
+        The list, tuple, or array of X values for the Histogram to plot from.
+
+    y : :class:`list`, :class:`tuple`, :class:`numpy.ndarray`
+        The list, tuple, or array of Y values for the Histogram to plot from.
+
+    graph_scene : Optional[:class:`.GraphScene`], optional
+        The :class:`.GraphScene` instance with which to align the Histogram's points.
+        When using Histogram in a GraphScene, this should be passed in as ``self``.
```

Comment:
I could see about removing that, yeah. I just figured it could be best to document how this should be done if scaling it. For the moment, I'll just add in the `:class:` thing I forgot there.